libraries/cli/src/config.cpp

```diff
@@ -247,6 +252,26 @@ bool Config::nodeConfigured() { return node_configured_; }
 
 FullNodeConfig Config::getNodeConfiguration() { return node_config_; }
 
+void Config::addDebugLoggingConfiguration(Json::Value& logging_config) {
+  constexpr std::string_view debug_config_json = R"({
+      "name": "debug",
+      "on": false,
+      "verbosity": "DEBUG",
+      "outputs": [
+        {
+          "type": "file",
+          "file_name": "debug/TaraxaDebug_N1_%m%d%Y_%H%M%S_%5N.log",
+          "rotation_size": 10000000,
+          "time_based_rotation": "0,0,0",
+          "format": "%ThreadID% %ShortNodeId% %Channel% [%TimeStamp%] %SeverityStr%: %Message%",
+          "max_size": 1000000000
+        }
+      ]
+    })";
+  auto debug_config = util::readJsonFromString(debug_config_json);
+  logging_config["configurations"].append(debug_config);
```

Comment:
what happens if someone already has "debug" profile in his config and we add different profile here with the same name ?

Reply:
Right now we will have 2 configs with "debug" name. So with `--log-configurations "debug"` both of them will be enabled